Modifying schema creation to use clojure/java.jdbc

BerryDaniel opened the pull request:
## Status
IN DEVELOPMENT

## Description
Adjustment to the below, but modifying schema creation to use clojure/java.jdbc

Adds ability to auto create schema if AUTO_MIGRATE is true (or on spinning up the dev environment from the repl)

Commit:
Modifying schema creation to use clojure/java.jdbc

mrcnc commented on server/src/spacon/db/conn.clj:
```diff
@@ -44,17 +44,19 @@
+(defn create-schema []
+  (log/debug "Creating schema if it doesnt exist")
+  (sql/execute! db-spec ["CREATE SCHEMA IF NOT EXISTS spacon"])
+  (repl/migrate (loadconfig)))
```
I think the create-schema function should only create the schema and nothing else.  So you should remove this line but add `(create-schema)` to the load config function, so it creates the schema before trying to return a datastore that depends on it.

mrcnc commented on server/project.clj:
```diff
@@ -6,6 +6,7 @@
   :dependencies [[org.clojure/clojure "1.9.0-alpha14"]
                  [org.clojure/core.async "0.3.442"]
                  [org.clojure/data.json "0.2.6"]
+                 [org.clojure/java.jdbc "0.7.0-alpha3"]
```
Don't think you need this b/c it's included as a dependency of the other db dependencies

Commit:
adjusted changes made by review

- create-schema function only creates schema
- removed addition of dependencies that were already added by other libs

mrcnc commented:
👍
